dvclive/report.py

```diff
@@ -17,21 +19,24 @@ def get_scalar_renderers(scalars_folder):
             data = parse_tsv(file)
             for row in data:
                 row["rev"] = "workspace"
-            rel = file.relative_to(scalars_folder).with_suffix("")
-            name = str(rel.as_posix())
-            properties = {"x": "step", "y": name}
-            renderers.append(VegaRenderer(data, name, **properties))
+            y = file.relative_to(scalars_folder).with_suffix("")
+            name = file.relative_to(scalars_folder.parent).with_suffix("")
```

Comment:
@dberenbaum 

> Can we reorganize the structure of the markdown output and images? Using the keras callback, I end up with a dvclive dir that looks like:
> 
> ```shell
> $ tree results # name of the dvclive dir
> results
> ├── eval
> │   ├── accuracy.png
> │   └── loss.png
> ├── report.md
> ├── scalars
> │   ├── eval
> │   │   ├── accuracy.tsv
> │   │   └── loss.tsv
> │   └── train
> │       ├── accuracy.tsv
> │       └── loss.tsv
> └── train
>     ├── accuracy.png
>     └── loss.png
> ```
> 
> Could all of the md-related info be in one dir?
 
 The `.png` images corresponding to each `.tsv` plot are now generated in the same path:
 
 ```
 results
├── report.md
└── scalars
   ├── eval
   │   ├── accuracy.png
   │   ├── accuracy.tsv
   │   ├── loss.png
   │   └── loss.tsv
   └── train
       ├── accuracy.png
       ├── accuracy.tsv
       ├── loss.png
       └── loss.tsv
```

Reply:
WDYT about putting the images into a `static` dir? That would be more consistent with `dvc plots` and keep `scalars` uncluttered.

Reply:
Updated